packages/protocol/releaseData/initializationData/release6.json

```diff
@@ -0,0 +1,4 @@
+{
+  "StableTokenBRL": ["Celo Brazilian Real", "cREAL", 18, "0x000000000000000000000000000000000000ce10", "1000000000000000000000000", "47304000", ["0xD533Ca259b330c7A88f74E000a3FaEa2d63B7972", "0x989e1a3B344A43911e02cCC609D469fbc15AB1F1"], ["2000000000000000000000000", "6000000000000000000000000"], "ExchangeBRL"],
```

Comment:
@yorhodes and I were wondering if maybe the best course of action would be to not use the release\<NUMBER\> convention here (leave that for core contract releases), and instead start using a different convention for these sorts of deployments. E.g. release-cREAL.json? The CI script that we were talking about before that implicitly depends on the current naming scheme would need to be changed, I think just hardcoding the current initializationData filename there would be fine.